internal/common/constant/constant.go

```diff
@@ -39,9 +42,12 @@ const (
 // EmptyString defines a constant for the empty string
 const EmptyString = ""
 
+// NameDelimiter defines character that is used to separate name parts
+const NameDelimiter = "-"
+
 // TrueString defines a constant for the True string
 const TrueString = "True"
 
-// VeleroBackupNamePrefix represents the prefix for the object name generated
-// by the NonAdminController
-const VeleroBackupNamePrefix = "nab"
+// MaximumNacObjectNameLength represents Generated Non Admin Object Name and
+// must be below 63 characters, because it's used within object Label Value
+const MaximumNacObjectNameLength = validation.DNS1123LabelMaxLength
```

Comment:
why redefine const?

Reply:
Because this is more generic now and rename is to reflect it's current purpose..

Reply:
It's used 8 times (2x in the code and 6x in tests).

Reply:
IIUC, what @mateusoliveira43 is trying to say is why not use the `validation.DNS1123LabelMaxLength` const directly.

Reply:
Yes, and my explanation is that this constant is used 8 times in the code (2 in the main code, 6 in tests), so it's much easier to modify it in one place if we ever need to adjust the length limit. I'm happy to use validation.DNS1123LabelMaxLength directly, but this is similar to other constants we use in the code. For example, why not just use "" instead of defining a TrueString constant, or use - directly instead of defining a delimiter? Similarly, why not use 63 directly instead of creating this constant?

The point is that defining MaximumNacObjectNameLength makes the code more manageable. We are setting a constraint specific to our application, which happens to align with an external one (the 63-character limit), but this allows us to easily modify it if needed in the future. By controlling our own constant, we avoid relying too heavily on the "upstream" constraint, which might change or have different context in the future.

Additionally, the constant comes with a comment explaining why this 63-character limit is relevant for NAC object names in our context. Without it, we'd need to repeat this explanation in every place where we use the raw constant, which could make the code harder to maintain and understand.